### DISCUSSION TITLE
どんな種類のお茶が好きですか？　What kind of tea do you like?

### DISCUSSION THREAD

**Discussion by Yuki-Ohishi** [NAIST-SE/AttractingContributors#516] [General]
どんな種類のお茶が好きですか？
What kind of tea do you like?

**Comment by Yuki-Ohishi** [comment]
https://github.com/Yuki-Ohishi/What-kind-of-tea-do-you-like-.git
URLはこちらです.

**Comment by ani-hovhannisyan** [comment]
Hi, I have added 10th tea in your project, please can you do it back to mine?
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git

**Comment by XC1996114514** [comment]
伊藤園のお茶は大好きでえす
